mlir-tensorrt/executor/lib/Runtime/Backend/Lua/Modules/TensorRT/TensorRTModule.cpp

```diff
@@ -369,6 +449,27 @@ static Status enqueueV3Wrapper(AllocTracker &tracker,
 
   MTRT_DBGF("%s", "enqueueV3 successful and inputs are consumed");
 
+  if (hasResults) {
+    int64_t* desc = reinterpret_cast<int64_t*>(*outputDescriptors);
+
+    // Number of results are known in advance.
+    int64_t descIndex = 0;
+    int64_t nbResults = desc[descIndex++];
+    for (int64_t i = 0; i < nbResults; ++i) {
+      auto allocator = static_cast<CustomTensorRTOuputAllocator *>(
+          outputAllocatorTracker.get(i));
+      desc[descIndex++] = allocator->mOutputDims.size(); // rank
+      desc[descIndex++] =
+          reinterpret_cast<uintptr_t>(allocator->mOutputPtr); // output pointer
+      std::copy_n(allocator->mOutputDims.data(), allocator->mOutputDims.size(),
+                  desc + descIndex); // Copy input shape
+      descIndex += allocator->mOutputDims.size();
+      std::vector<int64_t> strides(allocator->mOutputDims.size(), 1); // For now, use 1 strides.
```

Comment:
This definitely isn't right. That will cause runtime errors for anything that tries to load from this buffer. What is the stride conventions for data allocated by TensorRT? Is it canonical strides or are there are specific set of layouts that may be used?

Reply:
Looks like MLIR-TRT does not use `nvinfer1::setAllowedFormat` API which allows formats other than `nvinfer1::TensorFormat::kLINEAR`. So for all practical purposes, stride here is canonical.

Reply:
I think we can use `IExecutionContext::getTensorStrides` API to retrieve stride information.